src/modules/App/types.ts

```diff
@@ -82,3 +82,6 @@ export default interface AppProps {
   // Customer provided callbacks
   eventHandlers?: SBUEventHandlers;
 }
+
+export const CACHED_MESSAGE_TEMPLATES_TOKEN_KEY = 'sendbird_message_templates_token_key';
+export const CACHED_MESSAGE_TEMPLATES_KEY = 'sendbird_message_templates_key';
```

Comment:
how about moving it into `src/utils/consts.ts`?

Reply:
done!